src/api/v2/endpoint/chat.py

```
@@ -62,6 +97,35 @@ async def chat_endpoint(request: ChatRequest, db: AsyncSession = Depends(get_db)
         )
 
 
+@router.post("/chat/stream")
+async def chat_stream_endpoint(
+    request: ChatRequest, db: AsyncSession = Depends(get_db)
```

Ruff (0.15.5)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)